sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsLiveTests.cs

```diff
@@ -120,20 +121,26 @@ public void ThreadCGUD_ParticipantAUR_MessageGSU_NotificationT()
             Pageable<ChatParticipant> chatParticipants = chatThreadClient.GetParticipants();
             var chatParticipantsCount = chatParticipants.Count();
 
-            var newParticipant = new ChatParticipant(user4);
-            var newParticipant2 = new ChatParticipant(user5);
+            var newParticipant = new ChatParticipant(user4) { DisplayName = "user4" };
+            var newParticipant2 = new ChatParticipant(user5) { DisplayName = "user5" };
             chatThreadClient.AddParticipants(participants: new[] { newParticipant });
             AddChatParticipantsResult addChatParticipantsResult = chatThreadClient.AddParticipant(newParticipant2);
 
             Pageable<ChatParticipant> chatParticipantsAfterTwoAdded = chatThreadClient.GetParticipants();
             PageableTester<ChatParticipant>.AssertPagination(enumerableResource: chatParticipantsAfterTwoAdded, expectedPageSize: 2, expectedTotalResources: 5);
             var chatParticipantsAfterTwoAddedCount = chatParticipantsAfterTwoAdded.Count();
 
+            Pageable<ChatMessage> messagesAfterParticipantsAdded = chatThreadClient.GetMessages();
```

Comment:
I believe it is possible that GetMessages won't return the messages that are just sent as everything is asynchronous. If that is the case, this will be an unreliable test. @juancamilor is not the case for the messages?

Reply:
There are a bunch of http calls in between, that is how we went around that issue for messages and it has been reliable for the past months.  
We moved the getmessages code to the last possible time for assertions, so I think it is ok.

Reply:
Let's not add these now if we are not fully sure. We are just about to release, so it is better to keep the pipeline in a stable state

Reply:
This is not new @RezaJooyandeh , we had that already.

Reply:
The test existed, but `GetMessages` and verifying them seems to be newly added, is not it?

Reply:
It does not show here, but we have those calls above : Here is an example ... 
https://github.com/Azure/azure-sdk-for-net/pull/19220/files#diff-311d4d1711e88af2c1023b9cdc66a6388f648c344701232330eff7556d94b8b4L104

Reply:
Thank you for the extra info. I approved it as well.